shell/initialize/client.js

```diff
@@ -327,70 +226,6 @@ async function render(to, from, next) {
       return next();
     }
 
-    let instances;
-
-    // Call fetch hooks on components matched by the route.
-    await Promise.all(Components.map((Component, i) => {
-      // Check if only children route changed
-      Component._path = compile(to.matched[matches[i]].path)(to.params);
-      Component._dataRefresh = false;
-      const childPathChanged = Component._path !== _lastPaths[i];
-
-      // Refresh component (call fetch) when:
-      // Route path changed part includes current component
-      // Or route param changed part includes current component and watchParam is not `false`
-      // Or route query is changed and watchQuery returns `true`
-      if (this._routeChanged && childPathChanged) {
-        Component._dataRefresh = true;
-      } else if (this._paramChanged && childPathChanged) {
-        const watchParam = Component.options.watchParam;
```

Comment:
We don't use the undocumented `watchParam`. https://github.com/nuxt/website-v2/issues/1977